src/main/java/no/nav/familie/inntektsmelding/forespørsel/rest/OppdaterForespørslerRequest.java

```diff
@@ -0,0 +1,19 @@
+package no.nav.familie.inntektsmelding.forespørsel.rest;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+
+import no.nav.familie.inntektsmelding.typer.dto.AktørIdDto;
+import no.nav.familie.inntektsmelding.typer.dto.OrganisasjonsnummerDto;
+import no.nav.familie.inntektsmelding.typer.dto.SaksnummerDto;
+import no.nav.familie.inntektsmelding.typer.dto.YtelseTypeDto;
+
+public record OppdaterForespørslerRequest(@NotNull @Valid AktørIdDto aktørId,
```

Comment:
Trenger vi aktørId og ytelsetype når vi har saksnummer?

Reply:
Per nå så trenger vi det for å opprette nye forespørsler. Tenker du vi kanskje ikke trenger å lagre det?